src/NuGet.Indexing/NuGet.Indexing.csproj

```diff
@@ -190,7 +190,7 @@
       <Version>2.2.2</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Services.Entities">
-      <Version>4.4.5-master-2699519</Version>
+      <Version>4.4.5-dev-2939401</Version>
```

Comment:
We don't need this change now.

Reply:
Discussed offline--it appears that due to the fact that `4.4.5-master-2699519` is more recent than `4.4.5-dev-2939401` according to SemVer, these changes were necessary to force NuGet to resolve to the `4.4.5-dev-2939401` properly.